test/test_mediafile.py

```diff
@@ -141,6 +141,12 @@ def test_add_image_structure(self):
                       type=ImageType.composer)
         mediafile.images += [image]
         mediafile.save()
+        
+        image = Image(data=self.png_data, desc=u'the sortname of the composer',
+                      type=ImageType.composer_sort)
+        mediafile.images += [image]
+        mediafile.save()
```

Comment:
AFAICT, this is a test relating to images. I'm not sure `composer_sort` stuff is relevant here.